eva-accession-release/src/main/resources/copy_release_to_ftp.sh

```diff
@@ -144,39 +161,38 @@ do
   cp $INPUT_FOLDER/README_general_info.txt ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}
-  cp $INPUT_FOLDER/${assembly}/README_species_issues.txt ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}
-  cp $INPUT_FOLDER/${assembly}/${assembly}_merged_ids.vcf.gz ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}
-  cp $INPUT_FOLDER/${assembly}/${assembly}_merged_ids.vcf.gz.tbi ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}
-  cp $INPUT_FOLDER/${assembly}/${assembly}_current_ids.vcf.gz ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}
-  cp $INPUT_FOLDER/${assembly}/${assembly}_current_ids.vcf.gz.tbi ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}
-  cp $INPUT_FOLDER/${assembly}/${assembly}_deprecated_ids.txt.gz ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}
-  cp $INPUT_FOLDER/${assembly}/${assembly}_merged_deprecated_ids.txt.gz ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}
-  for taxonomy in `grep ${assembly} /tmp/assembly_to_taxonomy_map.txt | cut -f 2`
+  cd $INPUT_FOLDER/${assembly}
+  cp README_species_issues.txt ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}
+  cp ${assembly}_merged_ids.vcf.gz ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}
+  md5sum ${assembly}_merged_ids.vcf.gz > ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}/md5checksums.txt
+  cp ${assembly}_merged_ids.vcf.gz.tbi ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}
+  md5sum ${assembly}_merged_ids.vcf.gz.tbi >> ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}/md5checksums.txt
+  cp ${assembly}_current_ids.vcf.gz ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}
+  md5sum ${assembly}_current_ids.vcf.gz >> ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}/md5checksums.txt
+  cp ${assembly}_current_ids.vcf.gz.tbi ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}
+  md5sum ${assembly}_current_ids.vcf.gz.tbi >> ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}/md5checksums.txt
+  cp ${assembly}_deprecated_ids.txt.gz ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}
+  md5sum ${assembly}_deprecated_ids.txt.gz >> ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}/md5checksums.txt
+  cp ${assembly}_merged_deprecated_ids.txt.gz ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}
+  md5sum ${assembly}_merged_deprecated_ids.txt.gz >> ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}/md5checksums.txt
+  cd -
+
+  for taxonomy in `grep ${assembly} /tmp/assembly_to_taxonomy_map.txt | cut -f2`
   do
-    dbsnp_database_name=`grep " ${taxonomy}$" ${INPUT_FOLDER}/species_name_mapping.tsv | cut -f 4`
+    dbsnp_database_name=`grep "${taxonomy}$" ${INPUT_FOLDER}/species_name_mapping.tsv | cut -f4`
     if [ -z "${dbsnp_database_name}" ]
     then
       echo "Warning: taxonomy ${taxonomy} not found in ${INPUT_FOLDER}/species_name_mapping.tsv. Won't copy the unmapped_ids report for that taxonomy."
     else
-      cp ${UNMAPPED_VARIANTS_FOLDER}/${dbsnp_database_name}* ${INTERMEDIATE_FOLDER}/${ASSEMBLIES}/${assembly}
+      cd ${UNMAPPED_VARIANTS_FOLDER}
```

Comment:
do you really need to change directory here?

Reply:
Yes, if we don't the nfs path for the unmapped variants report would be present in the checksum file